### PR TITLE
Update WindNet to include Wind

### PR DESCRIPTION
**Pull request by jacobbieker**
# Pull Request

## Description

Updates WindNet to actually use wind power as the target and as an input.

Fixes #

## How Has This Been Tested?

Unit test

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: ocf_datapipes/training/windnet.py [hunk 1]
```diff
@@ -268,7 +221,9 @@ def construct_sliced_data_pipeline(
         fork_keys.update(set(f"nwp/{k}" for k in datapipes_dict["nwp"].keys()))
 
     # We don't need somes keys even if they are in the data dictionary
-    fork_keys = fork_keys - set(["topo", "nwp", "sensor", "hrv", "pv_future", "pv"])
+    fork_keys = fork_keys - set(
```
Comment (jacobbieker) [inline ocf_datapipes/training/windnet.py:223]: @dfulu Just to I understand, to use the correct key for the location datapipe, "wind", , should I have it be removed from this set, or in it?

File: ocf_datapipes/training/common.py [hunk 2]
```diff
@@ -159,12 +167,23 @@ def open_and_return_datapipes(
         pv_datapipe = OpenPVFromNetCDF(
             pv=configuration.input_data.pv
         ).add_t0_idx_and_sample_period_duration(
-            sample_period_duration=timedelta(minutes=5),
+            sample_period_duration=timedelta(minutes=5),  # TODO Fix as India is 15 minutely
```
Comment (dfulu) [inline ocf_datapipes/training/common.py:170]: There is a frequency entry in the config model which maybe we should use for this: https://github.com/openclimatefix/ocf_datapipes/blob/main/ocf_datapipes/config/model.py#L134

File: ocf_datapipes/training/windnet.py (hunk 1 above)
Comment (dfulu) [reply ocf_datapipes/training/windnet.py:223]: What you've done looks right, or at least consistant. This code could probably be clearer, but the `fork_keys` are the keys that we will need a copy of the location datapipe for. It looks like you aren't slicing the wind data by location further below(?) so it shouldn't be in `fork_set`

File: ocf_datapipes/training/common.py [hunk 3]
```diff
@@ -726,6 +767,37 @@ def slice_datapipes_by_time(
...
+
+        datapipes_dict["wind"] = datapipes_dict["wind"].select_time_slice(
+            t0_datapipe=get_t0_datapipe(None),
+            sample_period_duration=minutes(15),
+            interval_start=minutes(-conf_in.wind.history_minutes),
+            interval_end=minutes(0),
+            fill_selection=production,
+        )
+
+        # Dropout on the PV, but not the future PV
```
Comment (dfulu) [inline ocf_datapipes/training/common.py:789]: Maybe update these few comments that mention PV to keep it clean?

**Review by dfulu — APPROVED** [review]
Looks good

File: ocf_datapipes/training/common.py (hunk 2 above)
Comment (jacobbieker) [reply ocf_datapipes/training/common.py:170]: Ah good shout, I'll update it to that, and the other ones as well

**Pull request merged**